src/rust/dbv2/migrations/2024-02-07-144601_fees/up.sql

```diff
@@ -0,0 +1,33 @@
+-- Your SQL goes here
+CREATE TABLE aggregator.fees (
+    "time" TIMESTAMPTZ,
```

Comment:
It looks like we are calculating fees over the course of an hour, but that might not be immediately clear to someone querying this table from PostgREST

What about `"start_time_1hr_period"`?